orbit/pkg/update/flag_runner.go

```diff
@@ -295,7 +295,7 @@ func readFlagFile(rootDir string) (map[string]string, error) {
 			continue
 		}
 		// split each line by "="
-		str := strings.Split(line, "=")
+		str := strings.SplitN(line, "=", 2)
```

Comment:
What is this fixing? (does it require `orbit/changes`?)

Reply:
Previously the line was split on `=` and only the first two fields were returned. The problem was that the `vmodule` flag often contains many `=` as part of the flags (see example flag in the QA plan). This change stops orbit from discarding everything after the second `=` when parsing the flags

Reply:
and yeah you're right it should have an `orbit/changes` entry

Reply:
The orbit kept restarting in a loop and it took me a while to figure out why. It was because when parsing the flags it would fail to return the full line, think there was a change because the incoming and parsed flags didn't match, then restart

Reply:
For reference this is what the `osquery.flags` file looks like when you set `vmodule` in `command_line_options`:

```
--vmodule=init=1,packs=1,config=1,carve*=1,unified_log=1
```

Reply:
We only want to split on the first instance of `=`